zebra-script/src/lib.rs

```diff
@@ -451,13 +440,13 @@ mod tests {
         let verifier = super::CachedFfiTransaction::new(transaction, vec![output]);
 
         let input_index = 0;
-        let branch_id = Blossom
-            .branch_id()
-            .expect("Blossom has a ConsensusBranchId");
 
-        verifier.is_valid(branch_id, input_index + 1).unwrap_err();
-
-        verifier.is_valid(branch_id, input_index + 1).unwrap_err();
+        verifier
+            .is_valid(NetworkUpgrade::Blossom, input_index + 1)
+            .expect_err("verification should fail");
+        verifier
+            .is_valid(NetworkUpgrade::Blossom, input_index + 1)
+            .expect_err("verification should fail");
```

Comment:
```suggestion
        verifier
            .is_valid(NetworkUpgrade::Blossom, input_index + 1)
            .expect_err("verification should fail");
```

Reply:
This shouldn't have been merged. The verifier is supposed to fail twice. It's in the name of the test, similarly to other tests in the file.

Reply:
I'll fix it in another PR.

Reply:
sorry about that, i thought it was an oversight.